.travis.yml

```diff
@@ -20,7 +20,7 @@ env:
   - LANG=en_US.UTF-8
   matrix:
     - DESTINATION="platform=iOS Simulator,name=iPhone 7,OS=10.2" SDK="iphonesimulator10.2"
-    - DESTINATION="platform=iOS Simulator,name=iPhone 5s,OS=8.1" SDK="iphonesimulator8.1"
+    - DESTINATION="platform=iOS Simulator,name=iPhone 5,OS=8.1" SDK="iphonesimulator8.1"
```

Comment:
It possible / simple, consider noting that the iPhone 5 was chosen as it is a 32-bit device.